epa-under-siege.html

```diff
@@ -1,1741 +1,1704 @@
 ---
+layout: default
+title: "Part 1 - The EPA Under Siege"
+description: "The EPA Under Siege: Trump's Assault in History and Testimony"
+author: "Christopher Sellers, Lindsey Dillon, Jennifer Liss Ohayon, Nick Shapiro, Marianne Sullivan, Chris Amoss, Stephen Bocking, Phil Brown, Vanessa De la Rosa, Jill Harrison, Sara Johns, Katherine Kulik, Rebecca Lave, Michelle Murphy, Liza Piper, Lauren Richter, Sara Wylie"
 ---
-<html>
-    <head>
-    	<meta content="text/html; charset=UTF-8" http-equiv="content-type">
-    	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-    	<meta name="viewport" content="width=device-width, initial-scale=1">
-    	<title>Part 1 - The EPA Under Siege</title>
-    	<link rel="stylesheet" href="/assets/post.css">
-    	<link rel="stylesheet" href="/assets/tachyons.min.css"/>
-    	<link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
-        <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon">
-        <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
-        <meta property="og:title" content="The EPA Under Siege"/>
-        <meta property="og:type" content="article"/>
-        <meta property="og:url" content="{{ site.production_url }}{{ page.url | replace:'index.html',''}}"/>
-        <meta property="og:image" content="{{ site.production_url }}/assets/ch1-cover.jpg"/>
-        <meta property="og:description" content="Trump’s Assault in History and Testimony"/>
-        <meta name="twitter:card" content="summary_large_image">
-        <meta name="twitter:site" content="@EnviroDGI">
-        <meta property="article:author" content="Christopher Sellers, Lindsey Dillon, Jennifer Liss Ohayon, Nick Shapiro, Marianne Sullivan, Chris Amoss, Stephen Bocking, Phil Brown, Vanessa De la Rosa, Jill Harrison, Sara Johns, Katherine Kulik, Rebecca Lave, Michelle Murphy, Liza Piper, Lauren Richter, Sara Wylie"/>
-    </head>
-    <body class="bg-white">
-        <script
-			  src="https://code.jquery.com/jquery-3.2.1.min.js"
-			  integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-			  crossorigin="anonymous"></script>
-        <script src="jquery.sticky.js"></script>
-        <script type="text/javascript">
-          $(document).ready(function(){
-            $("#sticky-header").sticky({topSpacing:0, wrapperClassName:"header-wrapper"});
-        });
-        </script>
-        <header id="sticky-header" class="w-100 pv1">
-            <a href="/index.html" title="Home" class="">
-                <img id="nav-logo" src="assets/edgi-logo-b.png">
-            </a>
-            <div id="nav-txt" class="di"><h3 class="f5 ttu tl mb0 di">Part 1</h3><h3 class="f5 fw3 ttu tl mb0 di"> | The EPA Under Siege</h3></div>
-        </header>
 
-        <div class="mb4 dib ">
+<body class="bg-white">
```

Comment:
can we move the class/style stuff out of the body tags? as I think they really should be in the layout